typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -172,17 +202,18 @@ export default class Axes2DLayer extends Layer<Axes2DLayerProps> {
         min: number,
         max: number,
         viewSide: ViewSide,
-        pixel2world: number
+        pixel2world: number,
+        mv: number,
+        mh: number,
+        useExponential: boolean,
```

Comment:
I think it would be better to be able to override the number formatting, eg. with a callback. Say a callback that takes a number and returns a formatted number as a string. This way we could ensure consistent formatting in the map view, readout, and other representations of the data. A good default could be to use [d3-format](https://github.com/d3/d3-format)